src/site/_data/eleventyComputed.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+const outputPermalink = require('../../build/output-permalink');
+
+module.exports = {
+  permalink: function (data) {
+    if (Object.keys(data).length === 0) {
+      return;
+    }
+    const build = outputPermalink(data);
+    const serverless = path.join(
+      '/serverless',
+      build.replace('index.html', ''),
+    );
+
+    return {build, serverless};
```

Comment:
How does this work with serving this locally? Surely permalink is basically broken for normal development now?